index.js

```diff
@@ -195,6 +194,30 @@ var new_client = function(api_key, config) {
 
   }
 
+  client.all_flags = function(user, fn) {
+    var cb = fn || noop;
+    var _self = this;
+
+    if (this.stream && this.initialized) {
+      cb(null, Object.keys(_self.features).reduce(function(accum, current) {
+        accum[current] = evaluate(_self.features[current], user);
+        return accum;
+      }, {}));      
+    } 
+    else {
+      var request = make_request(client, '/api/eval/features');
+      request(function(response) {
+        features = response.getBody();
+        cb(null, Object.keys(features).reduce(function(accum, current) {
+          accum[current] = evaluate(features[current], user);
```

Comment:
I've been thinking about this. I think we should do the following:

* Add an optional `send_events` param on this method, with a default to `false`. If `send_events` is `true`, we queue up flag request events for everything in the list. 
* If the user is not sending us events automatically, we can expose a method to manually trigger flag request events.

The most common use case for this method is to bubble feature flags up to the user on the client side. In that case, users can take a bundle of all flags, but then selectively determine which flags are actually accessed.